build-tools-internal/src/main/groovy/elasticsearch.fips.gradle

```diff
@@ -18,6 +18,7 @@ if (BuildParams.inFipsJvm) {
     File fipsResourcesDir = new File(project.buildDir, 'fips-resources')
     boolean java8 = BuildParams.runtimeJavaVersion == JavaVersion.VERSION_1_8
     boolean zulu8 = java8 && BuildParams.runtimeJavaDetails.toLowerCase().contains("azul")
+    boolean oracle = BuildParams.runtimeJavaDetails.toLowerCase().contains('oracle')
```

Comment:
This reminded me that we can remove the zulu8 handling since we don't run Azul in FIPS anymore

Reply:
Good point. I removed zulu related stuff. Thanks!

Reply:
Do we even want to test FIPS for Java 8 anymore? I just noticed that [CI](https://github.com/elastic/elasticsearch/blob/7.x/.ci/matrix-runtime-javas-fips.yml) has it for only 11 and 16. But the PR FIPS job somehow runs Java 8 and it just failed [a bunch more](https://gradle-enterprise.elastic.co/s/qy7n6iuxwof32/tests/failed) for this PR (funny they didn't fail when the PR was initially raised). Not sure where the CI jobs for PR are defined these days. Should we update the definitions? I can also look into those failures closely and fix them. But that's only meaningful if we want to support Java 8. Thoughts?

Reply:
TBH I remmebered wre decided on 11,latest for master but not sure if this was intended for 7.x to or just an aggressive backport on my side. We have pinned 8 version to last known good working for adoptopenjdk so we could( should ) be using these

Reply:
These failures are now fixed by:
1. Resuing Zulu's BCJSSE security and policy files
2. Add SunRsaSign provider

This change works for the latest JDK 8, but not the earlier ones, i.e. prior to 272. Ideally we may want to keep the periodical FIPS CI for both old and new JDK 8 since they both their values. For the PR CI, I think it is fine to just stay using the latest JDK 8. So I'll merge this PR as is and work on the periodical CI in a separate PR. Please let me know if you have any concerns.

Reply:
> I think it is fine to just stay using the latest JDK 8. So I'll merge this PR as is 

:+1: 


> Ideally we may want to keep the periodical FIPS CI for both old and new JDK 8 since they both their values

Given recent discussions, I don't think a pre-272 JDK8 in CI is worth the effort/time.